<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>285</width>
    <height>175</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>AutoFarm</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59)</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="oresWarp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Варп с рудой</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oresSlot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Слот с киркой</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobsWarp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>Варп с мобами</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobsSlot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>Слот с оружием</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>265</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(214, 198, 255);
background-color: rgb(214, 198, 255);</string>
    </property>
    <property name="text">
     <string>СТАРТ</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>265</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);
color:rgb(0, 0, 0);
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string> %p%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid #ffffff;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Промежуток (мин)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="trayCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>115</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 2px solid rgb(226, 193, 255); 
color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Свернуть в трэй</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
